security/network_bound_disk_encryption/nbde-about-disk-encryption-technology.adoc

```diff
@@ -0,0 +1,30 @@
+// CNF-2127 assembly
+[id="nbde-about-disk-encryption-technology"]
+= About disk encryption technology
+include::modules/common-attributes.adoc[]
+:context: nbde-implementation
+
+toc::[]
+
+Network-Bound Disk Encryption (NBDE) allows you to encrypt root volumes of hard drives on physical and virtual
+machines without having to manually enter a password when restarting machines.
+
+include::modules/nbde-disk-encryption-technology-comparison.adoc[leveloffset=+1]
+
+include::modules/nbde-key-escrow.adoc[leveloffset=+2]
+
+include::modules/nbde-tpm-encryption.adoc[leveloffset=+2]
+
+include::modules/nbde-network-bound-disk-encryption.adoc[leveloffset=+2]
+
+include::modules/nbde-secret-sharing-encryption.adoc[leveloffset=+2]
+
+include::modules/nbde-using-tang-servers-for-disk-encryption.adoc[leveloffset=+1]
+
+include::modules/nbde-deciding-the-number-of-tang-servers-to-use.adoc[leveloffset=+2]
+
+include::modules/nbde-installation-scenarios.adoc[leveloffset=+1]
```

Comment:
My $0.02 is to put these installation scenarios near the top of the installation assembly.